server/GameController.js

```diff
@@ -98,13 +107,107 @@ module.exports = function(data) {
 	};
 
 	/*
+	update a users hand by replacing the used card with a new random one
+	*/
+	var updateHand = function(userId, usedCard) {
+		players.forEach(function(player){
+			if (player.uId === userId) {
+				player.hand.forEach(function(card){
+					if (card === usedCard) {
+						var index = Math.floor((Math.random() * whiteCards.length) + 1);
+						card = whiteCards[index];
+					}
+				});
+			}
+		});
+	};
+
+	/*
+		submit the answer and change the used card with another one in players
+	 */
+	var submitAnswer = function(playerId, answer, callback) {
+
+		var ans = {
+			playerId: playerId,
+			answerText: answer,
+			playersVote: []
+		};
 
+		var currentRound = rounds[rounds.length-1];
+		currentRound.answers.push(ans);
+		updateHand(playerId, answer);
+
+		//check if everyone submitted
+		if (currentRound.answers.length === players.length) {
+			callback({
+				answers: currentRound.answers
+			});
+		}
```

Comment:
maybe an else cause here, so we can track how many answer have been submitted, error handling etc